test/loader/pipeline.test.js

```diff
@@ -30,7 +30,7 @@ describe('pipeline loader', () => {
 
     // when
     const pipeline = loader(node, dataset, { context, variables, basePath: context.basePath })
-    await pipeline.initVariables()
+    await pipeline._pipeline.initVariables()
```

Comment:
Is this something clients would be typing as well, to get the inner `_pipeline` object?

Reply:
I just added `._pipeline` to have access to the inner object during debugging. I think that test should be even changed to init the variables by calling `.read()`, which will also trigger `.init()` + `.initVariables()`.

Reply:
I think we can live with that, if this is only for debugging or forcing the initial state in test